jsribosome.go

```diff
@@ -492,8 +495,12 @@ func jsProcessArgs(jsr *JSRibosome, args []Arg, oArgs []otto.Value) (err error)
 	return
 }
 
+const (
+	HolochainErrorPrefix = "Holochain Error"
+)
+
 func mkOttoErr(jsr *JSRibosome, msg string) otto.Value {
-	return jsr.vm.MakeCustomError("HolochainError", msg)
+	return jsr.vm.MakeCustomError(HolochainErrorPrefix, msg)
```

Comment:
By changing this constant from "HolochainError" to "Holochain Error" that will mean that anybody who adds the "returnError" config will also have to change their "isErr" function to catch this new error flag I believe. Is there a good reason to do it that outweighs that cost?

Reply:
Ok, I changed it back, however note that this code ALSO adds in isErr() to the api so you don't need to write it, and it uses the constant.